Resources/Prototypes/_NF/radio_channels.yml

```diff
@@ -0,0 +1,7 @@
+- type: radioChannel
+  id: Traffic
+  name: chat-radio-traffic
+  keycode: 'd'
+  frequency: 1022
+  color: "#CC6633"
+  longRange: true #DeltaV: Lets it be used farther than like 20 feet from station
```

Comment:
thats not what long range does:
1. it makes it not need a comms server
2. solar flares cant block it
3. it works across different maps (like station and centcom)

should this get removed, would simplify sop stuff too but needs adding to a comms server @Lyndomen